### DISCUSSION TITLE
🚀 Feature Request: allow auto approve d1 migrations apply

### DISCUSSION THREAD

**Discussion by shyim** [cloudflare/workers-sdk#13506] [Feature Requests]
### Describe the solution

I can apply migrations using `wrangler d1 migrations apply`, but I have to say yes to the prompt. It's annoying.

My workaround is:

```
yes | pnpm --prefix api run wrangler d1 migrations apply --local shopmonDB
```

**Comment by Ehbraheem** [comment]
@shyim If this is still a problem for you, you can always bypass the confirmation like below
```bash
yes Y | npx wrangler d1 migrations apply DB_NAME
```

I'll open a PR to make it easy to bypass the confirmation using a flag

**Comment by celestial-rose** [comment]
Hey there, any update for this ? Am I missing something or is this not critical to all of us ?

**Comment by Ehbraheem** [comment]
> Hey there, any update for this ? Am I missing something or is this not critical to all of us ?

@celestial-rose You can track the changes here #10598. It's approved now should be released this week.

**Comment by serhii-chernenko** [comment]
@Ehbraheem It's already been 2 months. How is it going?
